src/runtime/hexagon/rpc/android_bash.sh.template

```diff
@@ -20,7 +20,18 @@ export LD_LIBRARY_PATH=.
 
 # Enable FARF-based logging for Hexagon code invoked by 'tvm_rpc_android_server'.
 export ADSP_LIBRARY_PATH=`pwd`
-echo 0x1f > tvm_rpc_android.farf
+
+# Runtime log levels can be selectively enabled by computing a bitmask
+# corresponding to the levels you want to enable. These get forwarded to
+# logcat by the DSP RPC daemon. The bits for each level are:
+
+# 0x01 - Hexagon LOW / TVM DEBUG
+# 0x02 - Hexagon MEDIUM / TVM INFO
+# 0x04 - Hexagon HIGH / TVM WARN
+# 0x08 - Hexagon ERROR / TVM ERROR
+# 0x10 - Hexagon FATAL / TVM FATAL
+
+echo 0x1e > tvm_rpc_android.farf
```

Comment:
can you add this to the HexagonLauncher so we will be able to set the debug level at the python level?

Reply:
Added in the latest commit.